### PR TITLE
MEN-5478: QEMU images: Support setting Server IP in /etc/hosts

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline just in case :woman_judge:

File: meta-mender-qemu/scripts/docker/setup-mender-configuration.py [hunk 1]
```diff
@@ -73,6 +74,26 @@ def main():
...
+            remote_path="/etc/hosts",
+            rootfs=rootfs)
+        with open("hosts") as fd:
+            hosts = fd.readlines()
+        hosts.append("\n")
+        hosts.append("# Added by setup-mender-configuration.py\n")
+        server_host = args.server_url.removeprefix("http://").removeprefix("https://")
+        hosts.append(f"{args.server_ip} {server_host}\n")
+        with open("hosts", "w") as fd:
+            fd.writelines(hosts)
```
Comment (kacf) [inline meta-mender-qemu/scripts/docker/setup-mender-configuration.py:91]: You could simplify this by using `open("hosts", "a") as fd` instead.

**Review by kacf — APPROVED** [review]

File: meta-mender-qemu/scripts/docker/setup-mender-configuration.py (hunk 1 above)
Comment (lluiscampos) [reply meta-mender-qemu/scripts/docker/setup-mender-configuration.py:91]: Or course :sweat_smile: Thanks. I was biased by the `json` approach above.

**Comment by lluiscampos** [comment]
The integration pipeline does not really test this, so I won't rerun it after the force-push to save resources. Merging.

**Pull request merged**